setup.py

```diff
@@ -34,21 +34,20 @@
 # limitations under the License.
 # ============================================================================
 #
-import setuptools
+from pathlib             import Path
+from pyTooling.Packaging import DescribePythonPackageHostedOnGitHub
 
-with open("README.md", "r") as file:
-	long_description = file.read()
+gitHubNamespace =        "edaa-org"
+packageName =            "pyEDAA.IPXACT"
+packageDirectory =       packageName.replace(".", "/")
+packageInformationFile = Path(f"{packageDirectory}/__init__.py")
 
-setuptools.setup(
-	name="pyIPXACT",
-	version="0.1.6",
-	author="Patrick Lehmann",
-	author_email="Paebbels@gmail.com",
+DescribePythonPackageHostedOnGitHub(
+	packageName=packageName,
 	description="A Document-Object-Model (DOM) for IP-XACT files.",
-	long_description=long_description,
-	long_description_content_type="text/markdown",
-	url="https://github.com/Paebbels/pyIPXACT",
-	packages=setuptools.find_packages(),
+	gitHubNamespace=gitHubNamespace,
+	sourceFileWithVersion=packageInformationFile,
+	developmentStatus="pre-alpha",
 	classifiers=[
 		"License :: OSI Approved :: Apache Software License",
```

Comment:
Do not specify the license as classifier, it's done automatically.

Do not specify Python versions. It's done automatically.

Reply:
Done. Out of curiosity, where are is list of python versions defined?

Reply:
Here: https://github.com/pyTooling/pyTooling.Packaging/blob/main/pyTooling/Packaging/__init__.py?ts=2#L256

3.6 ... 3.10 like in Actions.

Reply:
Hmmm. The list of versions is overriden if provided. Is the list of classifiers overriden as well? Or are those appended to the default?